### PR TITLE
document tunnel flag

### LINKED ISSUE #223: tunnel mode needs to be documented
as we now have working tunnel mode, we should also have it in the docs

### PR DESCRIPTION
**Pull request by xh3b4sd**
fixes #223 

RFR @JosephSalisbury @Nesurion @zeisss

### CODE REVIEW

File: docs/tunneling.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Tunneling
+
+Inago provides ssh tunneling to remote hosts. That way we are able to operate
+on units running on remote machines from our local environment. Simply add the
+`--tunnel` flag like with `fleetctl`.
+
+Fetching the status of `mygroup` running at `my.remote.host` could work like
+that.
```
Comment (Nesurion) [inline docs/tunneling.md]: like this.

File: docs/tunneling.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+```
+inagoctl --tunnel=my.remote.host status mygroup
+```
+
+Updating `mygroup` running at `my.remote.host` could work like that. Note that
```
Comment (Nesurion) [inline docs/tunneling.md]: like this:

maybe move the "note" after the command?

File: docs/tunneling.md (hunk 1 above)
Comment (JosephSalisbury) [inline docs/tunneling.md]: :gb: could work like this:

Comment (JosephSalisbury) [inline docs/tunneling.md]: you want the colon, is what i mean

**Comment by JosephSalisbury** [comment]
little bit of grammar, looks good overall

**Comment by xh3b4sd** [comment]
Nah. Going to merge this. LGTM :trollface:

**Pull request merged**